### ISSUE TITLE
Tests failing on openSUSE

### ISSUE THREAD

**Issue by susnux** [mathics/Mathics#1487]
I try to build mathics on openSUSE (for packaging), building works fine, but when running the test suite I see this two tests failing:

> test/test_importexport.py::test_export - AssertionError: assert False
> test/test_nintegrate.py::test_nintegrate[NIntegrate[x^2 y^(-.5), {x,0,1},{y,0,1}]-1.-]

Here is the log of the failing tests: [fail-log.txt](https://github.com/mathics/Mathics/files/6796898/fail-log.txt)

I already tried to track down the issue, I tried to set `CI` env variable (the server building this package only has minimal software installed), but that only skips the first failing test, the second still fails. Is it known to fail, or does it indicate something does not work in my setup?

It does not matter which python version (tried 3.6, 3.7, 3.8 and 3.9) I use nor which version of the dependencies.
Here is a full log of a build on openSUSE Tumbleweed: [build-log.txt](https://github.com/mathics/Mathics/files/6796916/build-log.txt)

**Comment by rocky** [comment]
Thanks for undertaking to package on SUSE. 

I  think these are known flaky tests. I'll try to load up SuSE and see if I can fix some of these long-standing problems.

**Comment by rocky** [comment]
@susnux  I spend some time today to try to understand more.  The NIntegrate failure is triggered when scipy is not installed. I have opened issue https://github.com/mathics/Mathics/issues/1501 to track this separately. (Obviously this has nothing to do with failing on openSUSE).

As for the importexport test what is happening is that this test is trying to write something to the filesystem and that can fail. It is hard to tell all the conditions under which this can happen, but I have adjusted that test to skip for some reason it can't read the that it supposedly wrote (but may have not actually done so).

FYI, things are currently in flux but we hope to have a release of Mathics core and Mathics Django out in a couple of weeks or so.
